cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -76,7 +77,42 @@ func main(clusterID string) error {
 	}
 
 	fmt.Println(query.Build())
-	fmt.Println("\nLink to Web Console - \n", GetLinkToWebConsole(hcpCluster.DynatraceURL, since, base64.StdEncoding.EncodeToString([]byte(query.finalQuery))))
+	SearchQuery := map[string]interface{}{
+		"version": 0,
+		"data": map[string]interface{}{
+			"tableConfig": map[string]interface{}{
+				"visibleColumns": []string{"timestamp", "status", "content"},
+				"columnAttributes": map[string]interface{}{
+					"columnWidths": map[string]interface{}{},
+					"lineWraps": map[string]interface{}{
+						"timestamp": true,
+						"status":    true,
+						"content":   true,
+					},
+					"tableLineWrap": true,
+				},
+				"columnOrder": []string{"timestamp", "status", "content"},
+			},
+			"queryConfig": map[string]interface{}{
+				"query":     query.finalQuery,
+				"timeframe": map[string]interface{}{"from": fmt.Sprintf("now()-%vh", since), "to": "now()"},
+				"filter": map[string]interface{}{
+					"datatype": "logs",
+					"filters":  map[string]interface{}{},
+					"sort": map[string]interface{}{
+						"field":     "timestamp",
+						"direction": "desc",
+					},
+				},
+				"showDqlEditor": true,
+			},
+		},
+	}
+	mStr, err := json.Marshal(SearchQuery)
+	if err != nil {
+		return fmt.Errorf("failed to Marshal json %v", err)
+	}
+	fmt.Println("\nLink to Web Console - \n", GetLinkToWebConsole(hcpCluster.DynatraceURL, since, url.PathEscape(string(mStr))))
```

Comment:
The since variable does not seem to be used in new Logs app when you generate the query and rather is added above in query. Can this be refactored to ensure we donot use it

Reply:
We didn't change the entire URL format. We'd replace base64 encoded with json encoded. Thus the since value is adding to _GetLinkToWebConsole_ func

Reply:
This is the return value for the function - https://github.com/openshift/osdctl/blob/2ebcf8bdf736343d069e6ee043ced5c25d7eb96f/cmd/cluster/dynatrace/logsCmd.go#L60

If you check the current implementation, the since value even if passed in the URL will not be reflect the actual time frame for logs. This is done in the following change of your PR
```
"timeframe": map[string]interface{}{"from": fmt.Sprintf("now()-%vh", since), "to": "now()"},```

This means the value is unused even if we pass it to the function